flow/connectors/snowflake/snowflake.go

```diff
@@ -877,6 +877,25 @@ func (c *SnowflakeConnector) CreateTablesFromExisting(ctx context.Context, req *
 	}, nil
 }
 
+func (c *SnowflakeConnector) RemoveTableEntriesFromRawTable(
+	ctx context.Context,
+	req *protos.RemoveTablesFromRawTableInput,
+) error {
+	rawTableIdentifier := getRawTableIdentifier(req.FlowJobName)
+	for _, tableName := range req.DestinationTableNames {
+		_, err := c.execWithLogging(ctx, fmt.Sprintf("DELETE FROM %s.%s WHERE _PEERDB_DESTINATION_TABLE_NAME = '%s'"+
+			" AND _PEERDB_BATCH_ID > %d AND _PEERDB_BATCH_ID <= %d",
+			c.rawSchema, rawTableIdentifier, tableName, req.NormalizeBatchId, req.SyncBatchId))
```

Comment:
```suggestion
	for _, tableName := range req.DestinationTableNames {
		_, err := c.execWithLogging(ctx, fmt.Sprintf("DELETE FROM %s.%s WHERE _PEERDB_DESTINATION_TABLE_NAME = %s"+
			" AND _PEERDB_BATCH_ID > %d AND _PEERDB_BATCH_ID <= %d",
			QuoteIdentifier(c.rawSchema), QuoteIdentifier(rawTableIdentifier), QuoteLiteral(tableName), req.NormalizeBatchId, req.SyncBatchId))
```

Reply:
oh this is snowflake. We could use some escaping methods here. Or use parameters for string/number literals